### PR TITLE
Install additional R libaries

### PR DESCRIPTION
**Pull request by dominikl**
Installs the addtional R libraries

- [tidyverse](https://www.tidyverse.org/) - Various useful tools, especially ggplot2
- [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html) - Image analysis

in the docker image.

### CODE REVIEW

**Comment by jburel** [comment]
The notebook still works after adding the new libraries
We will need to list them somewhere so people do not have to read the Dockerfile to figure out

File: Dockerfile [hunk 1]
```diff
@@ -25,6 +25,9 @@ USER root
 RUN apt-get update && \
     apt-get -y install libssl-dev libxml2-dev libcurl4-openssl-dev libpcre3 libpcre3-dev liblzma-dev libbz2-dev libjpeg-dev libssh2-1-dev
 
+# Required for the additional R libraries (see bottom)
+RUN apt-get -y install libtiff-dev libpng-dev libfftw3-dev
```
Comment (joshmoore) [inline Dockerfile:29]: Probably better to include this in the previous line to save one Docker layer.

File: Dockerfile [hunk 2]
```diff
@@ -75,4 +78,7 @@ RUN Rscript install.R --version=v0.4.0
 # install r-kernel and make it accessible
 RUN Rscript -e "install.packages(c(\"devtools\"), repos = c(\"http://irkernel.github.io/\", \"http://cran.rstudio.com\"))"
 
-RUN Rscript -e "library(\"devtools\")" -e "install_github(\"IRkernel/repr\")" -e "install_github(\"IRkernel/IRdisplay\")" -e "install_github('IRkernel/IRkernel')" -e "IRkernel::installspec()"
+RUN Rscript -e "library(\"devtools\")" -e "install_github(\"IRkernel/repr\")" -e "install_github(\"IRkernel/IRdisplay\")" -e "install_github('IRkernel/IRkernel')" -e "IRkernel::installspec()" 
+
+# install additional useful R libraries
+RUN Rscript -e "install.packages(\"tidyverse\")" -e "source(\"https://bioconductor.org/biocLite.R\")" -e "biocLite(\"EBImage\")"
```
Comment (joshmoore) [inline Dockerfile:84]: ditto

**Comment by jburel** [comment]
@dominikl will you have time to work on this PR? I have done some work on CellProfiler for Elmi and it will conflict

**Comment by dominikl** [comment]
I'll close it for now, can just rebase and reopen later again.

**Pull request closed without merging**